### ISSUE TITLE
Implement Logger

### ISSUE THREAD

**Issue by orkaboy** [orkaboy/FFX_TAS_Python#1]
Continue to convert messages to Log(level, ...) format. Need to complete a run or two to test that nothing breaks and that the log levels work as intended.

Prerequisites: The config yaml is merged into main so that log levels can be set in the config file.

**Comment by orkaboy** [comment]
Need to implement in accordance with feedback from Discord.
